src/config/client_config.cc

```diff
@@ -30,32 +30,35 @@
 #include "util/upnp_quirks.h"
 
 ClientConfig::ClientConfig()
-    : clientInfo(std::make_shared<ClientInfo>())
 {
-    clientInfo->name = "Unknown";
-    clientInfo->type = ClientType::Unknown;
-    clientInfo->flags = QUIRK_FLAG_NONE;
-    clientInfo->matchType = ClientMatchType::None;
-    clientInfo->match = "";
+    auto cInfo = ClientInfo {
+        "Unknown",
+        ClientType::Unknown,
+        QUIRK_FLAG_NONE,
+        ClientMatchType::None,
+        "",
+    };
+    clientInfo = std::make_shared<ClientInfo>(cInfo);
```

Comment:
What about a `move` here to avoid copying cInfo?

Reply:
I think I tried this and found that adding std::move made no difference to the compiled size. Will need to look at it again.

Reply:
Yeah no difference to size. Added move anyway. clang-tidy doesn't complain so it should be fine.